tests/framework/di/ContainerTest.php

```diff
@@ -100,4 +104,103 @@ public function testDefault()
         $qux = $container->get('qux', [3, ['a' => 4]]);
         $this->assertEquals(4, $qux->a);
     }
+
+    public function testInvoke()
+    {
+        $this->mockApplication([
+            'components'=>[
+                'qux' => [
+                    'class' => 'yiiunit\framework\di\stubs\Qux',
+                    'a' => 'belongApp',
+                ]
+            ]
```

Comment:
We need a test for the case, when we have two components of the same class with different names:

```php
                'qux1' => [
                     'class' => 'yiiunit\framework\di\stubs\Qux',
                     'a' => 'belongAppQux1',
                 ]
```

and the following tests of arguments:

```php
function (stubs\QuxInterface $qux1)
```

```php
function (stubs\QuxInterface $fooName)
```